lib/client.rb

```diff
@@ -22,6 +23,10 @@ def get_replicas_info
   def fail_over
     @client.sentinel("failover", @master_name)
   end
+
+  def get_master_address
+      @client.sentinel("get-master-addr-by-name", @master_name)
+    end
```

Comment:
(cosmetic) can remove extra white space before `end` to make it aligned with `def`

Reply:
addressed this change